pkg/resolver/searchComplete.go

```diff
@@ -74,6 +81,13 @@ func (s *SearchCompleteResult) searchCompleteQuery(ctx context.Context) {
 			//Adding notNull clause to filter out NULL values and ORDER by sort results
 			whereDs = append(whereDs, goqu.L(`"data"->>?`, s.property).IsNotNull())
 		}
+		//RBAC CLAUSE
+		if s.userData != nil {
+			whereDs = append(whereDs,
+				buildRbacWhereClause(ctx, s.userData)) // add rbac
+		} else {
+			panic("RBAC clause is required!")
```

Comment:
Do we have any context that we could add here to help us debug if we panic here? I'm thinking something like the user ID, the input query, or anything else that could help us later.